src/cmake/dependencies/CMakeLists.txt

```diff
@@ -91,13 +103,22 @@ if(WIN32 OR WIN64)
     #We need to rename libcurl_a.lib to libcurl.lib for find_package(CURL)
     file(RENAME ${CMAKE_CURRENT_BINARY_DIR}/install/lib/libcurl_a.lib ${CMAKE_CURRENT_BINARY_DIR}/install/lib/libcurl.lib)
 endif()
+
+  
+endif()
   
 endif()
 
 ##############
 ##  wxWidgets  ##
 ##############
 if(BUILD_wxWidgets)
+
+#Check if CURL is not already available
```

Comment:
faut-il écrire 
`#Check if wxWidgets is not already available`

à la place de 
`#Check if CURL is not already available`
?

Reply:
Oui. Le commentaire a été modifié